src/main/java/importDeck/ImportDeckInteractor.java

```diff
@@ -90,6 +90,9 @@ public void importDeck(ImportDeckInputData inputData){
         try {
             ImportDeckDsOutputData dsOutputData = dsGateway.importFromFile(dsInputData);
             String deckName = dsOutputData.getImportedDeckName();
+            if (Deck.getTracker().get(deckName) != null){
+                deckName = deckName + " - Copy";
```

Comment:
Might get overly long. Instead of " - Copy" lets just do "+"

Reply:
Sure